core/benches/unprocessed_packet_batches.rs

```diff
@@ -64,10 +58,7 @@ fn build_randomized_packet_batch(
                     1,
                     recent_blockhash.unwrap_or_else(Hash::new_unique),
                 );
-                let mut packet = Packet::from_data(None, tx).unwrap();
-                let sender_stake = distribution.sample(&mut rng);
-                packet.meta_mut().sender_stake = sender_stake as u64;
-                packet
+                Packet::from_data(None, tx).unwrap()
```

Comment:
nit: after removed code to set random sender_stake, this function became same as `build_packet_batch()`. Can remove `build_randomized_packet_batch()` and related bench test all together. This can be done in follow up PR.